### PR TITLE
OHRM5X-1477: Test Automation : Leave - Leave period

### PR DESCRIPTION
**Pull request by OvaraPerera**

### CODE REVIEW

File: src/test/functional/cypress/integration/feature/Leave/leavePeriod.spec.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    it('Create db snapshot for Reset Leave Period', function () {
+      cy.loginTo(user.admin, '/leave/defineLeavePeriod');
+      cy.wait('@retriveLeavePeriod');
+      cy.getOXD('form').within(() => {
+        cy.getOXDInput('Start Month').selectOption('May');
+        cy.getOXDInput('Start Date').selectOption('15');
+        cy.getOXD('button').contains('Save').click();
+      });
+      cy.wait('@updateLeavePeriod');
+      //cy.toast('success', 'Successfully Saved');
```
Comment (RajithaKumara) [inline src/test/functional/cypress/integration/feature/Leave/leavePeriod.spec.js:97]: no commented codes. is this pending assertion

File: src/test/functional/cypress/integration/feature/Leave/leavePeriod.spec.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    it('Verify the Current Leave Period and End date is getting updated', function () {
+      cy.loginTo(user.admin, '/leave/defineLeavePeriod');
+      cy.wait('@retriveLeavePeriod');
+      cy.getOXD('form').within(() => {
+        cy.getOXDInput('Start Month').selectOption('June');
+        cy.getOXDInput('Start Date').selectOption('28');
+        cy.getOXD('button').contains('Save').click();
+      });
+      cy.wait('@updateLeavePeriod');
+      //cy.toast('success', 'Successfully Saved');
```
Comment (RajithaKumara) [inline src/test/functional/cypress/integration/feature/Leave/leavePeriod.spec.js:58]: no commented codes

File: src/test/functional/cypress/integration/feature/Leave/leavePeriod.spec.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  describe('save leave period', function () {
+    it('update the leave period to any day of the year', function () {
+      cy.loginTo(user.admin, '/leave/defineLeavePeriod');
+      cy.wait('@retriveLeavePeriod');
+      cy.getOXD('form').within(() => {
+        cy.getOXDInput('Start Month').selectOption('May');
+        cy.getOXDInput('Start Date').selectOption('15');
+        cy.getOXD('button').contains('Save').click();
+      });
+      //cy.toast('success', 'Successfully Saved');
```
Comment (RajithaKumara) [inline src/test/functional/cypress/integration/feature/Leave/leavePeriod.spec.js:46]: no commented codes

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

**Comment by OvaraPerera** [comment]
Closed as a new PR was added.
New PR Link- https://github.com/orangehrm/orangehrm/pull/1338

**Pull request closed without merging**